src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,107 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.ValidationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String FIRST_VALID_LOGIN = "Username";
+    private static final String SECOND_VALID_LOGIN = "Username2";
+    private static final String VALID_PASSWORD = "Password";
+    private static final String INVALID_LOGIN = "12345";
+    private static final String INVALID_PASSWORD = "12345";
+    private static final int VALID_AGE = 18;
```

Comment:
I consider constants in tests a bad pattern

simply put - it makes your test methods less readable, you always need to scroll to find a corresponding value
apart from that, it makes your test method depend on a single constant, which gets even worse, if you use that constant in several places

while constants are ok for production code, they're not ok for test code :)

I recommend reading this [article](https://medium.com/@erik.sacre/on-tdd-literals-versus-constants-in-tests-8ac992c7873f)